openapi.yaml

```diff
@@ -5611,6 +5624,47 @@ components:
       example:
         - href: 'https://example.openeo.org/docs/errors/SampleError'
           rel: about
+    log_level:
+      description: |-
+        The severity level of the log entry.
+
+        The order of the levels is as follows (from low to high severity): `debug`, `info`, `warning`, `error`.
+        
+        The level `error` usually stops processing the data.
```

Comment:
I'm not sure if it's useful to claim that the "error" level corresponds with termination. There could still be retry attempts after an error situation. Some systems also include the level "critical" for errors that cause termination.

Reply:
While it looks like an addition to this PR, this has been in the API definition before, see line 5925. It also just says that it "usually" stops, which I think it true in most cases.

Reply:
Maybe my problem is that "The level `error` usually stops processing the data." sound like the logging _itself_ makes the processing stop.

Bit more verbose proposal:

> The level `error` usually corresponds with critical issues that unexpectedly terminate the normal data processing.

Reply:
Ah, now I understand better what the issue is. That makes sense, will update.